launcher_go/v2/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/alecthomas/kong"
+	"github.com/discourse/discourse_docker/launcher_go/v2/utils"
+	"golang.org/x/sys/unix"
+	"os"
+	"os/exec"
+	"os/signal"
+)
+
+type Cli struct {
+	Version      kong.VersionFlag `help:"Show version."`
+	ConfDir      string           `default:"./containers" help:"Discourse pups config directory." predictor:"dir"`
+	TemplatesDir string           `default:"." help:"Home project directory containing a templates/ directory which in turn contains pups yaml templates." predictor:"dir"`
```

Comment:
I'm also not sure if we should make this configurable as the help text is only useful for users that understand how `discourse_docker` is structured. I think we should just simplify things and assume that `launcher` has to be ran from the root path of the `discourse_docker` repository.

Reply:
the default values assumes that it's run in the root path... so you're suggesting we remove the option entirely? I defaulted to making any file/directory have sane defaults but be configurable.

Reply:
> but be configurable.

Do we have any use cases where this needs to be configurable because I'm not sure if launcher is used anywhere else except for the root path of the repository? My preference now will be to just ship the v1 without making it configurable at all.

Reply:
I've updated the option so it's now hidden from CLI help output